mmpose/codecs/udp_heatmap.py

```diff
@@ -163,10 +163,7 @@ def decode(self, encoded: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
                 if k % 3 == 0:
                     # for classification map
                     ks = 2 * self.blur_kernel_size + 1
-                else:
```

Comment:
The code can be simplified like:

```python
for cls_heatmap in heatmaps[::3]:
    ks = 2 * self.blur_kernel_size + 1
    cv2.GaussianBlur(cls_heatmap, (ks, ks), 0, cls_heatmap)
```

Reply:
Sure. The accuracy has also been verified.